src/CLIFramework/Config/GlobalConfig.php

```diff
@@ -46,4 +51,15 @@ public function isDebug()
         }
         return $this->isDebug;
     }
+
+    /**
+     * Returns the directory of pid files.
+     */
+    public function getPidDirectory()
+    {
+        if (isset($this->config['core']['pid_dir'])) {
+            $this->pidDir = $this->config['core']['pid_dir'];
```

Comment:
Can we simply return the configure value directly? since we don't provide interface for user to set the pid directory from method call:

```php
return $this->config['core']['pid_dir'];
```